modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeCatchAllCheckTest.java

```diff
@@ -0,0 +1,129 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.processor;
+
+import com.liferay.portal.kernel.util.ListUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.SourceFormatterArgs;
+import com.liferay.source.formatter.check.UpgradeCatchAllCheck;
+
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+/**
+ * @author Kyle Miho
+ */
+@RunWith(Parameterized.class)
+public class UpgradeCatchAllCheckTest extends BaseSourceProcessorTestCase {
+
+	@Parameterized.Parameters(name = "Testcase-{index}: Testing {0}")
+	public static Iterable<Object[]> data() throws Exception {
+		String issueKey = System.getProperty("issue.key", null);
```

Comment:
Let's treat `"issue.key"` as a comma-delimited String, so that multiple issue keys can be passed at once.

Reply:
Done